conf/application.conf

```diff
@@ -40,11 +40,6 @@ play.filters.enabled = [
   filters.SessionIdFilter
 ]
 
-location.canonical.list {
-  all = "location-autocomplete-canonical-list.json"
-  nonUK = "location-canonical-list-nonUK.json"
-}
-
```

Comment:
Any reason why this has been removed? I suspect we'll need this for the country input on non UK address pages

Reply:
it was duplicated further down